projects/1.20/assets/create-connected/create-connected/lang/zh_cn.json

```
@@ -0,0 +1,244 @@
+{
+    "advancement.create_connected.brass_gearbox": "精心安排",
+    "advancement.create_connected.brass_gearbox.desc": "放下一个黄铜十字齿轮箱",
+    "advancement.create_connected.control_chip": "精密制造",
+    "advancement.create_connected.control_chip.desc": "组装一个控制芯片",
+    "advancement.create_connected.overpowered_brake_0": "压倒性的动力",
+    "advancement.create_connected.overpowered_brake_0.desc": "使用被红石激活的制动器时仍然保持动力网络高速运行\n§7（隐藏进度）",
+    "advancement.create_connected.overstress_clutch": "断路器",
+    "advancement.create_connected.overstress_clutch.desc": "触发过应力离合器",
+    "advancement.create_connected.pulse_generator_infinite_loop": "无限循环",
+    "advancement.create_connected.pulse_generator_infinite_loop.desc": "编入一个故障程序使可编程脉冲生成器过载\n§7（隐藏进度）",
+    "advancement.create_connected.root": "欢迎来到机械动力：创意传动",
+    "advancement.create_connected.root.desc": "适用于所有情况的小玩意儿",
+    "advancement.create_connected.sequenced_pulse_generator": "算力至上",
```

原文Computational Supremacy可能引自量子计算领域的Quantum (Computational) Supremacy（eg. https://arxiv.org/abs/1203.5813 ），（大致）用以指设想中通过量子计算超越现有的所有经典计算手段。在网络上查找到某些资源将这一概念译作“量子霸权”，不过我个人觉得这译名有点奇怪了。当然，作者也可能压根没往这个方面想（x

就本例而言，supremacy似乎是指Sequenced Pulse Generator而不是抽象概念，意思应该是Sequenced Pulse Generator有更强的编程、运算能力。

```suggestion
    "advancement.create_connected.sequenced_pulse_generator": "至高算力",
```

---

这样翻译感觉也可以 Supremacy在群星里的翻译也是至高